modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointPagesWriter.java

```diff
@@ -183,18 +184,24 @@ private IgniteConcurrentMultiPairQueue<PersistentPageMemory, FullPageId> writePa
 
         Result<PersistentPageMemory, FullPageId> queueResult = new Result<>();
 
+        GroupPartitionId partitionId = null;
+
         while (!shutdownNow.getAsBoolean() && writePageIds.next(queueResult)) {
             beforePageWrite.run();
 
-            FullPageId fullId = queueResult.getValue();
+            FullPageId pageId = queueResult.getValue();
```

Comment:
"fullId" was better. When I see a pageId, I think that it lacks groupId, as it should, because usually it's a "long" value.

Reply:
Fix it.